### PR TITLE
WINC-921: reduce periodic alert frequency

### PR DESCRIPTION
**Pull request by alinaryan**
Reduces the community-4.12 and community-4.13 periodic frequency
from every four hours M-F to twice per week on Tues and Fri. Also adjusts
the job to not report on success to reduce noise in the #forum-winc Slack channel.

### CODE REVIEW

**Comment by alinaryan** [comment]
Went with once a day Tuesday through Friday based on this [comment](https://github.com/openshift/release/pull/36514#discussion_r1110218620) in the last PR.

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.12__okd.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ tests:
   container:
     from: src
 - as: aws-e2e-operator
-  cron: 0 */4 * * 1-5
+  cron: 0 0 * * 2-5
```
[the remark this answers is not in the document]
Comment (sebsoto) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.12__okd.yaml:56]: This [comment](https://github.com/openshift/release/pull/36514#discussion_r1110218620) is saying tues AND friday not tues through friday

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

**Review by aravindhp — APPROVED** [review]
/lgtm

**Pull request merged**